src/main/java/com/codessquad/qna/exception/NoQuestionException.java

```diff
@@ -0,0 +1,7 @@
+package com.codessquad.qna.exception;
+
+public class NoQuestionException extends RuntimeException{
```

Comment:
```suggestion
public class NoQuestionException extends RuntimeException {
```
포매팅 아직 잘 안되네요~ 신경 많이 써주셔야 할 것 같아요.